tests/files/complete.export.snapshot.yaml

```
@@ -0,0 +1,266 @@
+    "actions": [],
+    "format": "SODIMM",
+    "model": "48594D503131325336344350362D53362020",
+    "interface": "DDR3",
+    "type": "RamModule",
+    "manufacturer": "Hynix Semiconductor",
+    "serialNumber": "4F43487B",
+    "speed": 667.0
+  },
+  {
+    "type": "HardDrive",
+    "model": "HTS54322",
+    "size": 238475,
+    "interface": "ATA",
+    "serialNumber": "E2024242CV86HJ",
+    "manufacturer": "Hitachi",
+    "actions": [
+    {
+      "type": "BenchmarkDataStorage",
+      "elapsed": 16,
+      "writeSpeed": 21.8,
+      "readSpeed": 66.2
+    },
+    {
+      "type": "TestDataStorage",
+      "length": "Extended",
+      "elapsed": 2,
+      "severity": "Error",
+      "status": "Unspecified Error. Self-test not started."
+    },
+    {
+      "type": "EraseBasic",
+      "steps": [
+      {
+        "type": "StepRandom",
+        "startTime": "2018-07-03T09:15:22.257059+00:00",
+        "severity": "Info",
+        "endTime": "2018-07-03T10:32:11.843190+00:00"
+      }
+      ],
+      "startTime": "2018-07-03T09:15:22.256074+00:00",
+      "severity": "Info",
+      "endTime": "2018-07-03T10:32:11.848455+00:00"
+    }
+    ]
+  },
+  {
+    "size": 160041.88569599998,
+    "variant": "1A01",
+    "actions": [
+      {
+        "type": "EraseBasic",
+        "steps": [
+          {
+            "type": "StepRandom",
+            "endTime": "2019-10-23T08:35:31.400587+00:00",
+            "severity": "Info",
+            "startTime": "2019-10-23T07:49:54.410830+00:00"
+          }
+        ],
+        "endTime": "2019-10-23T08:35:31.400988+00:00",
+        "severity": "Error",
+        "startTime": "2019-10-23T07:49:54.410193+00:00"
+      },
+      {
+        "elapsed": 22,
+        "writeSpeed": 17.3,
+        "readSpeed": 41.6,
+        "type": "BenchmarkDataStorage"
+      },
+      {
+        "status": "Completed without error",
+        "reallocatedSectorCount": 0,
+        "currentPendingSectorCount": 0,
+        "assessment": true,
+        "severity": "Info",
+        "offlineUncorrectable": 0,
+        "lifetime": 4692,
+        "type": "TestDataStorage",
+        "length": "Short",
+        "elapsed": 118,
+        "powerCycleCount": 5293
+      }
+    ],
+    "model": "WDC WD1600BEVT-2",
+    "interface": "ATA",
+    "type": "DataStorage",
+    "manufacturer": "Western Digital",
+    "serialNumber": "WD-WX11A80W7430"
+  },
+  {
+    "actions": [
+      {
+        "writeSpeed": 17.1,
+        "type": "BenchmarkDataStorage",
+        "elapsed": 22,
+        "readSpeed": 41.1
+      },
+      {
+        "type": "EraseSectors",
+        "startTime": "2019-08-19T16:48:19.689794+00:00",
+        "steps": [
+          {
+            "startTime": "2019-08-19T16:48:19.690458+00:00",
+            "type": "StepRandom",
+            "severity": "Info",
+            "endTime": "2019-08-19T17:34:22.930562+00:00"
+          }
+        ],
+        "severity": "Info",
+        "endTime": "2019-08-19T17:34:22.930959+00:00"
```

There is no Action Erase with more than one step. We can add it as follows:

```suggestion
        "steps": [
          {
            "startTime": "2019-08-19T16:48:19.690458+00:00",
            "type": "StepRandom",
            "severity": "Info",
            "endTime": "2019-08-19T17:34:22.930562+00:00"
          },
          {
            "startTime": "2019-08-19T17:34:22.690458+00:00",
            "type": "StepZero",
            "severity": "Info",
            "endTime": "2019-08-19T18:34:22.930562+00:00"
          }
        ],
        "severity": "Info",
        "endTime": "2019-08-19T18:34:22.930959+00:00"
      }
```